site/docs/how-to-release.md

```diff
@@ -435,10 +435,10 @@ spark-shell \
 
 To verify using Flink, start a Flink SQL Client with the following command:
 ```bash
-wget ${MAVEN_URL}/iceberg-flink-runtime/{{ icebergVersion }}/iceberg-flink-runtime-{{ icebergVersion }}.jar
+wget ${MAVEN_URL}/iceberg-flink-runtime-${FLINK_VERSION}/{{ icebergVersion }}/iceberg-flink-runtime-${FLINK_VERSION}-{{ icebergVersion }}.jar
```

Comment:
Same question here about `FLINK_VERSION`. I would add a note indicating the user has to define the variables.